fix: tutorial (#43)
algolia/algoliasearch-netlify: docs/schema.md

# Extracted Record Schema

Algolia is a schema-less search engine. However to provide a simple and easy experience with Neltify and the Crawler,
we currently populate Algolia with a standard record.

> All root level properties are a computation of multiple selectors with fallback. We **reserve the possibility to change the extraction logic** or to add more properties, but **not remove top property without proper deprecation period**.

Reviewer: ```suggestion
All root level properties are a computation of multiple selectors with fallback. We **might change the extraction logic** or to add more properties, but **will not remove root level properties without a proper deprecation period**.
```
